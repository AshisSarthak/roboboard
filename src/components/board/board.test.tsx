import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Board from "./board";

test("renders APP", () => {
  const { asFragment } = render(<Board dimension={5} />);
  expect(asFragment()).toMatchSnapshot();
});
