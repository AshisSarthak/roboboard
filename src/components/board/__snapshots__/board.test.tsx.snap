// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders APP 1`] = `
<DocumentFragment>
  <div
    class="board-container MuiBox-root css-0"
  >
    <div
      class="board-box MuiBox-root css-87jy3b"
    >
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="grid-boxes MuiBox-root css-1qpslqa"
      />
      <div
        class="direction-arrow robot MuiBox-root css-vh7mn0"
        data-testid="robot-box"
      >
        <img
          alt=""
          class="north"
          src="robot-vacuum-cleaner.png"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
